docs/core/tools/dotnet-workload-update.md

```diff
@@ -62,8 +62,6 @@ For more information about the `dotnet workload` commands, see the [dotnet workl
 
 [!INCLUDE [no-cache](../../../includes/cli-no-cache.md)]
 
-[!INCLUDE [sdk-version](../../../includes/cli-sdk-version.md)]
-
 [!INCLUDE [source](../../../includes/cli-source.md)]
```

Comment:
Remove source and temp-dir includes?

Reply:
Those were just moved in the synopsis, they're still valid options.